### PR TITLE
Add tests and git hook

### PR DESCRIPTION
**Pull request by mecm1993**
This is related to issue #28.

**New dependencies**:
 - [pre-commit](https://www.npmjs.com/package/pre-commit)
 - [enzyme & enzyme-adapter-react-16](https://www.npmjs.com/package/enzyme)
 - [enzyme-to-json](https://www.npmjs.com/package/enzyme-to-json)
 - [cross-env](https://www.npmjs.com/package/cross-env)

### CODE REVIEW

File: src/__tests__/CoderCard.test.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { shallow } from 'enzyme'
+
+import { CoderCard } from '../components/CoderCard'
+
+describe('CoderCard', () => {
+  let wrapper
+  
+  const coder1 = {
```
Comment (GianCastle) [inline src/__tests__/CoderCard.test.js]: Hi!

Please, move the mock data into a single array of object on another file.

Comment (mecm1993) [reply src/__tests__/CoderCard.test.js]: Hi,

I created a new file in utils for the mock data.

**Comment by mecm1993** [comment]
ping @GianCastle

**Comment by GianCastle** [comment]
Hi, sorry for the delay, bro!

Could you please update this PR? It has some MC

**Comment by mecm1993** [comment]
> Hi, sorry for the delay, bro!
> 
> Could you please update this PR? It has some MC

You got it 💯

**Pull request merged**